File: 1165.cpp
#include <iostream>

using namespace std;

int main(){
    int n, valores=0, aux=0, qntdDivisoes=0;
    cin >> n;
    while (aux < n){
        cin >> valores;
        for (int i=1;i<=valores;i++){
            if (valores % i == 0){
                qntdDivisoes += 1;
            }
        }
        if (qntdDivisoes == 2){
            cout << valores << " eh primo" << endl;
        } else {
            cout << valores << " nao eh primo" << endl;
        }
        aux += 1;
        qntdDivisoes = 0;
    }
    return 0;
}

File: 1165.py
N = int(input())
for i in range(1, N+1):
    X = int(input())
    divisoes = 0
    for j in range(1, X+1):
        if X % j == 0:
            divisoes += 1
    if divisoes > 2:
        print(f'{X} nao eh primo')
    else:
        print(f'{X} eh primo')